### PR TITLE
add mariadb clients

### PR DESCRIPTION
**Pull request by rolandknight**
add mariadb clients (mariadb and mariadb-dump)

### CODE REVIEW

**Comment by alecthomas** [comment]
Are there no Mac packages available?

**Comment by rolandknight** [comment]
Official Mac packages aren't available anymore:
https://mariadb.com/kb/en/mariadb-pkg/#comment_3703

**Comment by alecthomas** [comment]
Sorry for the delay, I was pondering whether we want to do this, but I think it's okay.

File: mariadb-clients.hcl [hunk 1]
```diff
@@ -0,0 +1,10 @@
+description = "Mariadb client."
```
Comment (alecthomas) [inline mariadb-clients.hcl:1]: Can you please rename it to `mariadb-client.hcl`, so it's consistent with the existing `mysql-client.hcl`.

**Comment by alecthomas** [comment]
Thanks!

**Pull request merged**